### ISSUE TITLE
Default Box IP is not reachable

### ISSUE THREAD

**Issue by feurigel** [ion2s-gmbh/vbox#29]
The default ip `10.0.0.1` in the box.sample.yml is a bad example, because it is not reachable. If I start the box with this ip I get this warning message:

```
==> vbox-dev: You assigned a static IP ending in ".1" to this machine.
==> vbox-dev: This is very often used by the router and can cause the
==> vbox-dev: network to not work properly. If the network doesn't work
==> vbox-dev: properly, try changing this IP.
```

Better is an example like `10.0.0.101`. It doesn't throw an error and it works. :-)

Tested with: Windows 10 Version: 1903, vbox 1.12.0

**Comment by sebbaum** [comment]
Thanks, @feurigel for reporting this issue.

Feel free to submit a PR otherwise I will fix it in the next release.

Cheers
Sebastian

**Comment by sebbaum** [comment]
This has been fixed in release 1.13.0

Cheers
Sebastian
